F21/cemilcan98/app.py

```diff
@@ -0,0 +1,84 @@
+def toLowerCase(i):
+    if(i[1] in ["en", "en-US", "en-IE", "en-Latn"]):
+        y = i[0].lower()
+        return y
+
+    # Checking for the special letters in Turkish and Azeri 
+    if(i[1] == "tr" or i[1] == "az"):

```

Comment:
Would it be better to have elif's throughout since these cases are mutually exclusive?